sos/report/plugins/ostree.py

```diff
@@ -17,6 +17,9 @@ class OSTree(Plugin, IndependentPlugin):
     profiles = ('system', 'sysmgmt', 'packagemanager')
     files = ('/ostree',)
     services = ('ostree-finalize-staged', 'ostree-boot-complete')
+    option_list = [
+        PluginOpt('fsck', default=False, desc='collect ostree fsck')
```

Comment:
Unittests are failing since `PluginOpt` is not imported in this plugin.

Reply:
I was blind, thanks.